content/en/docs/installing-on-linux/public-cloud/install-ks-on-azure-vms.md

```diff
@@ -0,0 +1,208 @@
+    master:
+    - master-0
+    - master-1
+    - master-2
+    worker:
+    - node000000
+    - node000001
+    - node000002
+```
+For a complete configuration sample explanation, please see [this file](https://github.com/kubesphere/kubekey/blob/master/docs/config-example.md).
+
+### Configure the Load Balancer
+
+In addition to the node information, you need to provide the load balancer information in the same yaml file. For the IP address, you can find it in *Azure -> KubeSphereVMRG -> PublicLB*. Assume the IP address and listening port of the **load balancer** are `40.81.5.xx` and `6443` respectively, and you can refer to the following example.
+
+#### The configuration example in config-sample.yaml
+
+```yaml
+## Public LB config example
+## apiserver_loadbalancer_domain_name: "lb.kubesphere.local"
+  controlPlaneEndpoint:
+    domain: lb.kubesphere.local
+    address: "40.81.5.xx"
+    port: "6443"
+```
+
+> - Note we are using the public LB directly instead of an Internal LB due to the Azure [Load Balancer limits](https://docs.microsoft.com/en-us/azure/load-balancer/load-balancer-troubleshoot#cause-4-accessing-the-internal-load-balancer-frontend-from-the-participating-load-balancer-backend-pool-vm).
+
+### Persistent Storage Plugin Configuration
+
+See [Storage Configuration](../storage-configuration) for details.
+
+### Configure the Network Plugin
+
+Azure Virtual Network doesn't support IPIP mode which used by [calico](https://docs.projectcalico.org/reference/public-cloud/azure#about-calico-on-azure). So let's change the network plugin to flannel.
+
+```yaml
+  network:
+    plugin: flannel
+    kubePodsCIDR: 10.233.64.0/18
+    kubeServiceCIDR: 10.233.0.0/18
+```
+
+### Start to Bootstrap a Cluster
+
+After you complete the configuration, you can execute the following command to start the installation:
+
+```bash
+./kk create cluster -f config-sample.yaml
+```
+
+Inspect the logs of installation:
+
+```bash
+kubectl logs -n kubesphere-system $(kubectl get pod -n kubesphere-system -l app=ks-install -o jsonpath='{.items[0].metadata.name}') -f
+```
+When the installation finishes, you can see the following message:
+
+```bash
+#####################################################
+###              Welcome to KubeSphere!           ###
+#####################################################
+Console: http://10.128.0.44:30880
+Account: admin
+Password: P@88w0rd
+NOTES：
+  1. After logging into the console, please check the
+     monitoring status of service components in
+     the "Cluster Management". If any service is not
+     ready, please wait patiently until all components
+     are ready.
+  2. Please modify the default password after login.
+#####################################################
+https://kubesphere.io             2020-xx-xx xx:xx:xx
+```
+
+### Access KubeSphere Console
+
+Now that KubeSphere is installed, you can access the web console of KubeSphere by the following URL.
+
+-  http://{LoadBalancer IP}:30880
```

Comment:
`Now that KubeSphere is installed, you can access the web console of KubeSphere by the following URL.
-  http://{LoadBalancer IP}:30880`

=>
Congratulation! Now you can access the KubeSphere console using `http://10.128.0.44:30880` (Replace the IP with yours).